src/Castle.Core.Tests/CustomModifiersTestCase.cs

```diff
@@ -194,6 +193,8 @@ public void ReflectionReturnsCorrectModoptOnReturnTypeForGeneratedType(string ty
 			Assume.That(CustomModifiersTestCase.customModifiers.ContainsKey(typeNameWithoutSuffix));
 
 			var modopts = this.generatedTypes[typeName].GetMethod("Foo").ReturnParameter.GetOptionalCustomModifiers();
+			Assume.That(modopts.Length > 0); // If this fails on mono/linux we have to revisit the commits and issues for IL method parameter modifiers. https://github.com/castleproject/Core/issues/277
```

Comment:
> IL method parameter modifiers

The last two tests are for custom modifiers on the return type, probably just write "custom modifiers".